src/test/java/com/hotline/tests/BaseTest.java

```diff
@@ -60,17 +67,82 @@ public Boolean isVisibleInViewport(WebElement element) {
         return wait.until((ExpectedCondition<Boolean>) driver -> {
             assert driver != null;
             return (Boolean) ((JavascriptExecutor) driver).executeScript(
-                    "var elem = arguments[0],                 " +
-                            "  box = elem.getBoundingClientRect(),    " +
-                            "  cx = box.left + box.width / 2,         " +
-                            "  cy = box.top + box.height / 2,         " +
-                            "  e = document.elementFromPoint(cx, cy); " +
-                            "for (; e; e = e.parentElement) {         " +
-                            "  if (e === elem)                        " +
-                            "    return true;                         " +
-                            "}                                        " +
-                            "return false;                            "
-                    , element);
+                "var elem = arguments[0],                 " +
+                    "  box = elem.getBoundingClientRect(),    " +
+                    "  cx = box.left + box.width / 2,         " +
+                    "  cy = box.top + box.height / 2,         " +
+                    "  e = document.elementFromPoint(cx, cy); " +
+                    "for (; e; e = e.parentElement) {         " +
+                    "  if (e === elem)                        " +
+                    "    return true;                         " +
+                    "}                                        " +
+                    "return false;                            "
+                , element);
         });
     }
+
+    protected void checkElementsContent(List<WebElement> elements, String expectedData) {
+        for (WebElement i : elements) {
+            Assert.assertTrue(i.getText().contains(expectedData));
+        }
+    }
+
+    protected List<WebElement> sortElementsByRising(List<WebElement> sortedlist) {
+        Comparator<WebElement> comparator = Comparator.comparing(WebElement::getText);
+        sortedlist.sort(comparator);
+        return sortedlist;
+    }
+
+    protected List<WebElement> sortElementsByReducing(List<WebElement> sortedlist) {
+        Comparator<WebElement> comparator = Comparator.comparing(WebElement::getText).reversed();
+        sortedlist.sort(comparator);
+        return sortedlist;
+    }
+
+    protected List<Integer> sortIntegers(List<Integer> integerList) {
+        Comparator<Integer> comparator = Comparator.comparing(integer -> integer);
+        integerList.sort(comparator.reversed());
```

Comment:
In my opinion, creating a variable for only one use is redundant (at least in this case), so you can refactor it like this:
`integerList.sort(Comparator.comparing(integer -> integer));`